packages/backend/Dockerfile.prod

```diff
@@ -59,4 +59,4 @@ COPY --from=builder /home/rafiki/packages/backend/dist ./packages/backend/dist
 COPY --from=builder /home/rafiki/packages/token-introspection/dist ./packages/token-introspection/dist
 COPY --from=builder /home/rafiki/packages/backend/knexfile.js ./packages/backend/knexfile.js
 
-CMD ["node", "/home/rafiki/packages/backend/dist/index.js"]
+CMD ["node", "-r", "/home/rafiki/packages/backend/dist/telemetry/index.js", "/home/rafiki/packages/backend/dist/index.js"]
```

Comment:
Because auto-instrumentation essentially needs to wrap over multiple modules, everything needs to be loaded as the very first thing before app startup:

https://opentelemetry.io/docs/languages/js/getting-started/nodejs/#setup

otherwise, auto-instrumentation will not wok.

Reply:
We could just turn this CMD call `pnpm start` or something, but maybe its ok like this?

Reply:
just curious, what happens if we dont include the `--require` option? some error? silently fails? Or did it just coincidentally work before?

dont have a very strong opinion on the pnpm start command vs this but leaning towards this. since we may only want to start it this way in this context as opposed to Dockerfile.dev, from host machine, etc. Could always have different start commands in that case I guess though.

Reply:
It just "fails" silently, and doesn't generate traces. For example, if the pg module gets loaded before this postgres auto instrumentation, it just won't pick up those traces. 

If you uncomnent the debugger in the telemetry file, you'd be able to see those warnings.